exchange/exchangetest/eidpermissions-allowed-alias.json

```diff
@@ -61,6 +61,7 @@
                                 "foo": "appnexus"
                             },
                             "data": {
+                                "bidders": null,
```

Comment:
Why does it need to be specified as null?

Reply:
@SyntaxNode 
This is how this structure is defined:
```
type BidderConfig struct {
	Bidders []string `json:"bidders,omitempty"`
	Config  *Config  `json:"config,omitempty"`
}
```
`Bidders` is not a pointer, I think this is the reason. Do you think this should be modified?

Reply:
Ok fixed! 
Added `Bidders  []string  `json:"bidders,omitempty"``